config/routes.rb

```diff
@@ -205,6 +205,8 @@
       end
     end
 
+    resources :search, only: :index
+
```

Comment:
Do we want to use `:show` or `:create` rather than `:index`? It seems to me `:index` represents a collection of objects, whereas in our use case we're moreso getting a single search result set based on a query string. I'm not strongly opinionated either way, just a consideration in order to stay close to REST conventions

Reply:
Yeah, IMO we want to use `index` as the endpoint is ultimately returning a collection of search results.

Reply:
Cool, I'm good with that. We're not really doing many operations on RESTful resources with vets-api as-is, so it's not like we're diverging from an established pattern. :+1: